src/app/events/page.js

```diff
@@ -4,10 +4,12 @@ import CalendarEvents from "@/components/events/CalendarEvents";
 const Events = () => {
   return (
     <>
-      <div className="flex justify-center items-center bg-winc-pink w-full p-8 outline outline-8 outline-winc-pink outline-offset-8 my-4">
+      <div className="flex justify-center items-center bg-winc-pink w-full p-4 outline outline-8 outline-winc-pink outline-offset-4 my-4">
         <Title text="Programs and Events" textcolor="text-winc-white" />
       </div>
-      <CalendarEvents />
+      <div className="pt-16">
```

Comment:
we will do this padding in the calendar events component, not here creating another div